tests/cases/conformance/salsa/typeFromPropertyAssignment20.ts

```diff
@@ -0,0 +1,20 @@
+// @allowJs: true
+// @checkJs: true
+// @noEmit: true
+// @Filename: bluebird.js
+
+!function outer (f) { return f }(
```

Comment:
Could this be made simpler? Why are we applying identity and then negating it?

Reply:
(I'm doing some digging to find out why all these pieces appear to be required.)